src/components/shared/ItemImage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { GatsbyImage } from "gatsby-plugin-image"
+import { IGatsbyImageData } from "gatsby-plugin-image/dist/src/components/gatsby-image.browser"
+import Box from "@mui/material/Box"
+import { BoxProps } from "@mui/material"
+
+type ItemImageProps = {
+  image: IGatsbyImageData
+  alt: string
+}
+
+export const ItemImage = ({
+  image,
+  alt,
+  ...rest
+}: ItemImageProps & BoxProps) => (
+  <Box
+    maxWidth={{ xs: `73px`, lg: `247px` }}
+    width="100%"
+    height={{ xs: `41px`, lg: `140px` }}
+    borderRadius={{ xs: `4px`, lg: `8px 0 0 8px` }}
+    overflow="hidden"
+    style={{
+      transform: `translateZ(0)`,
+    }}
+    {...rest}
+  >
+    <GatsbyImage style={{ height: `100%` }} image={image} alt={alt} />
```

Comment:
Che ne pensi se lasciassimo che il component accetti `children` (le `BoxProps` lo prevedo già) e lasciassimo che sia il componente in cui si importa a passare l'immagine che vogliamo usarlo ?
In questo modo lo generalizziamo e funziona con tutte le immagini

Reply:
yes ci sta, provvedo